src/sentry/api/serializers/models/organization_member.py

```diff
@@ -1,21 +1,65 @@
 from collections import defaultdict
+from typing import Any, List, Mapping, MutableMapping, Optional, Sequence, Set
 
 from sentry import roles
 from sentry.api.serializers import Serializer, register, serialize
-from sentry.models import ExternalUser, OrganizationMember, OrganizationMemberTeam, Team, TeamStatus
+from sentry.models import (
+    ExternalUser,
+    OrganizationMember,
+    OrganizationMemberTeam,
+    Team,
+    TeamStatus,
+    User,
+)
+
+
+def get_serialized_users_by_id(users_set: Set[User], user: User) -> Mapping[str, User]:
+    serialized_users = serialize(users_set, user)
+    return {user["id"]: user for user in serialized_users}
+
+
+def get_team_slugs_by_organization_member_id(
```

Comment:
Is this code intended to be in this pr? Just checking since the title implies it's just adding types

Reply:
Yeah I wanted to move this code into a function so that it could have a more description name than `result` and so that I could annotate types.